SDK/AppCenterAuth/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common/Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common.csproj

```diff
@@ -13,7 +13,7 @@
     <RootNamespace>Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common</RootNamespace>
     <AssemblyName>Microsoft.AppCenter.Auth.Android.Bindings.Msal.Common</AssemblyName>
     <FileAlignment>512</FileAlignment>
-    <TargetFrameworkVersion>v8.1</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
```

Comment:
Is it still the case?

Reply:
At first build attempt with mono 5.4 yes.
Then I upgraded CI to mono 5.12 only later.
It does not matter though, I kept it like that to align with existing modules.